Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -224,6 +224,8 @@
     auto *progress = (WKModernDownloadProgress *)m_progress;
     [progress startUpdatingDownloadProgress];
 
+    send(Messages::DownloadProxy::StartUpdatingProgress());
```

Comment:
Should we e.g. use the name `DidStartUpdatingProgress`, since this message does not actually start updating the progress?